pyccel/ast/numpyext.py

```diff
@@ -982,6 +1003,9 @@ def is_elemental(self):
     'mod'       : NumpyMod,
     'float32'   : NumpyFloat32,
     'float64'   : NumpyFloat64,
+    'bool': NumpyBool,
+    'int8': NumpyInt8,
+    'int16': NumpyInt16,
```

Comment:
If you don't mind, could you please align the `:` signs to all the others above and below?